packages/flutter_localizations/lib/src/l10n/material_ja.arb

```diff
@@ -6,7 +6,7 @@
   "openAppDrawerTooltip": "ナビゲーションメニューを開く",
   "backButtonTooltip": "戻る",
   "closeButtonTooltip": "閉じる",
-  "nextMonthTooltip": "前の月へ",
+  "nextMonthTooltip": " 次の月へ",
```

Comment:
Extra space here? (or is that part of the word?)

Reply:
Good catch! That's me switching input methods in vim...